ui.apps/src/main/content/jcr_root/apps/saas-aem-module/i18n/en.json

```diff
@@ -1,6 +1,7 @@
 {
   "com.valtech.aem.saas.core.search.loadmore.button.label": "Load more",
   "com.valtech.aem.saas.core.search.submit.button.label": "Search",
-  "com.valtech.aem.saas.core.search.suggestion.text": "Did you mean: <em>{0}</em>?",
-  "com.valtech.aem.saas.core.search.input.placeholder.text": "Type search term here..."
+  "com.valtech.aem.saas.core.search.suggestion.text": "Did you mean: {0}",
```

Comment:
@jerome-mottet-valtech is this suggestion text with a placeholder fine for FE?

Reply:
`Did you mean:` should be enough. Unless the templating functionality is needed ? (it's not implemented in the FE yet)

Reply:
Usage of a placeholder is neat for cases when the text is in format like "Foo bar {0} buz".
I think that for this case I can remove it, since the suggestion is always at the end.

Reply:
Depends always on the language. Think or Arabic Languages with LTR...
Anyhow, this can be changed in a new version as well in the future if required